TODO.md

```diff
@@ -52,12 +52,21 @@ v0.7 testing
 * [x] [BUG] updating operator spec to scale all deployments down works, but resume doesn't
 
 v0.8 bugfixes
-* [ ] Verify that system works without ResourcePolicy set
+* [x] Verify that system works without ResourcePolicy set
 * [x] make `scaleStatePendingPeriod` configurable
-* [ ] profile slow reconcile (15s for ~300 deployments)
+* [x] profile slow reconcile (15s for ~300 deployments)
 * [x] Fix statuses after the change in scaling logic (scale based on lag) 
     
-v0.9
+v0.9 Multi-partition assignment
+* [x] Static multi-partition assignment
+* [ ] Consider making number of partitions optional in the spec
+
+v
+* [ ] Dynamic multi-partition assignment. Instead of static `numPartitionsPerInstance`:
```

Comment:
@hagen1778 @kvap what do you think about this approach? not for this release, but for next one

Reply:
This is a good idea. It will allow the konsumerator to assign fewer partitions to a consumer, if they are heavy.
Needs to be designed carefully.

Reply:
Yeah, this will complicate the logic **a lot**.

Reply:
Not sure if it is possible/reliable to do with k8s operator.

Reply:
It is definitely possible, reliability depends on implementation. As Constantin said - it needs to be designed carefully.